src/hotspot/share/memory/metaspaceShared.cpp

```diff
@@ -1356,6 +1356,8 @@ void MetaspaceShared::preload_and_dump(TRAPS) {
 
     HeapShared::init_for_dumping(THREAD);
 
+    // create a dummy manifest to cause more classes loaded
```

Comment:
How about: `// exercise the manifest processing code to ensue classes used by CDS are always archived`?

Reply:
s/ensue classes/ensure classes/